### ISSUE TITLE
screen refresh during credential input / slow afterwards 

### ISSUE THREAD

**Issue by tstreibl** [nextcloud/registration#175]
My nextcloud 15.07 is running on a raspberry pi under debian/apache/PHP 7.3.2. Background tasks are updated via the "Ajax" setting.

This setting results in problems when user tries to input his new credentials after confirming the link in the email invitation. During credential input the screen refreshes and clears the data the user typed into login or password fields. In my case this is so fast that it's nearly impossible to fill in both fields. In the beginning I thought that's a conflict with the unsplash app, nicely showing a new background picture on each page refresh - but after deactivating unsplash nothing changed.

However when I manage to fill in the credentials it takes forever until the user is forwarded to his new nextcloud account or any other system response can be seen. This is especially a problem when a password input was too short and the system requests a new, longer password. Althougt the new credentials are already accepted (I can see that the new user is already in the system, when I use a chrome browser in parallel to firefox, where I'm logged in as admin) the system still seems to wait for further input (up to 5 minutes!). When I change during this waiting time to a new tab in firefox the new user gets logged in smoothly into his new account, whereas the "old" tab is still "on hold".

I tested what happens when I change to "cron job" for the background job update. This solves the "screen refresh during credential input" problem. The other problem, that nextcloud doesn't respond in acceptable time after credential login, still remains.

**Comment by nickvergessen** [comment]
Feel free to reopen the issue when this is still happening on a recent Nextcloud version.
